security/subsystem/src/main/java/org/jboss/as/security/SecuritySubsystemParser.java

```diff
@@ -158,14 +162,20 @@ public void readElement(XMLExtendedStreamReader reader, List<ModelNode> list) th
                     parseProperties(Element.VAULT_OPTION.getLocalName(), reader, vault, VaultResourceDefinition.OPTIONS);
                     break;
                 }
+                case ELYTRON_INTEGRATION: {
+                    // the elytron-integration element was added in version 2.0 of the subsystem.
+                    if (schemaVer.compareTo(Namespace.SECURITY_2_0) < 0) {
```

Comment:
I have a really strong dislike for these within the XML parsers - the problem is they don't give the ability for something to be removed again in the future.